lib/identity/stateTransitions/assetLock/proof/instant/validateInstantAssetLockProofStructureFactory.js

```diff
@@ -51,6 +74,19 @@ function validateInstantAssetLockProofStructureFactory(
       return result;
     }
 
+    const outPointBuffer = transaction.getOutPointBuffer(rawAssetLockProof.outputIndex);
+    const outPointExists = await stateRepository.checkAssetLockTransactionOutPointExists(
```

Comment:
Let's use `verifyChainLockOutPointIsNotUsed`, and rename the method itself from `verifyChainLockOutPointIsNotUsed` to something like `checkAssetLockTransactionOutPointAlreadyUsed`